examples/triggerservice-image-update.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: standard.oam.dev/v1alpha1
+kind: TriggerService
+metadata:
+  name: image-rebase-trigger
+  namespace: default
+spec:
+  triggers:
+    - source:
+        # source is all the kpack Image resources in all the namespaces
+        type: resource-watcher
+        properties:
+          apiVersion: kpack.io/v1alpha2
+          # kpack needs to be installed on the cluster to have this resource type
+          kind: Image
+          events:
+            - update
+
+      # only trigger action when an Image is successfully rebased
+      filter: >
+        context.data.status.latestBuildReason == "STACK" && context.data.status.conditions[0].status == "True"
+      
+      action:
+        type: image-update-task
+        properties:
+          cmd: [/bin/sh, -c, "echo Image: ${SOURCE_NAME} in namespace: ${SOURCE_NAMESPACE} has been successfully rebased at $(date)"]
```

Comment:
Thanks for the contribution!
But I don't quite get the usage of this `image-update-task` job. It seems like in your example, this action will only echo a message in a created job, am I understanding it correctly?

Reply:
Yes, I just added added a simple echo command as an example 😬
The `image-update-task` definition is the more important part.
I could update the example if you any suggestions though.

Reply:
I see! In general, the `image-update-task` will just use job to handle some tasks. Why not we change the name from `image-update-task` to `task`?